root: clean up README (#16286)
goauthentik/authentik: README.md

## What is authentik?

authentik is an open-source Identity Provider (IdP) for modern SSO. It supports SAML, OAuth2/OIDC, LDAP, RADIUS, and more, designed for self-hosting from small labs to large production clusters.

> An [enterprise offering](https://goauthentik.io/pricing?utm_source=github) is available for organizations looking to replace or consolidate existing IdPs (e.g., Okta/Auth0, Entra ID, Ping Identity) at scale.

Reviewer: ```suggestion
Our [enterprise offering](https://goauthentik.io/pricing?utm_source=github) is available for organizations to securely replace or consolidate existing IdPs such as Okta, Auth0, Entra ID, and Ping Identity for robust, large-scale identity management.
```

> Our [enterprise offering](https://goauthentik.io/pricing?utm_source=github) is available for organizations to securely replace or consolidate existing IdPs such as Okta, Auth0, Entra ID, and Ping Identity for robust, large-scale identity management.

Reviewer 2: ~utm~

## Installation

> - Docker Compose: recommended for small/test setups. See the [documentation](https://docs.goauthentik.io/docs/install-config/install/docker-compose/?utm_source=github).

Reviewer 3: I think we lose the important bits here... explaining at beginnig of sentence which is best for which env. However.. overall it looks better so ok. ;-)

- Kubernetes (Helm Chart): recommended for larger setups. See the [documentation](https://docs.goauthentik.io/docs/install-config/install/kubernetes/?utm_source=github) and the Helm chart [repository](https://github.com/goauthentik/helm).

> - AWS CloudFormation: deploy on AWS using our official templates. See the [documentation](https://docs.goauthentik.io/docs/install-config/install/aws/?utm_source=github).

Reviewer 2: ~utm~
If we mention AWS CF here, should we also link to DO droplet? https://marketplace.digitalocean.com/apps/authentik

| ![](https://docs.goauthentik.io/img/screen_apps_light.jpg)  | ![](https://docs.goauthentik.io/img/screen_apps_dark.jpg)  |
| ![](https://docs.goauthentik.io/img/screen_admin_light.jpg) | ![](https://docs.goauthentik.io/img/screen_admin_dark.jpg) |

## Development and contributions

> See the [Developer Documentation](https://docs.goauthentik.io/docs/developer-docs/?utm_source=github) for information about setting up local build environments, testing your contributions, and our contribution process.

Reviewer 2: ~utm~

> ## Development

Reviewer: ```suggestion
## Development and contributions
```

> See the [Developer Documentation](https://docs.goauthentik.io/docs/developer-docs/?utm_source=github) and [CONTRIBUTING.md](CONTRIBUTING.md).

Reviewer: ```suggestion
See the [Developer Documentation](https://docs.goauthentik.io/docs/developer-docs/?utm_source=github).
```

No need to link to `contributing.md` when it just links to developer documentation.

## Security

Please see [SECURITY.md](SECURITY.md).

> ## Adoption and Contributions

Reviewer: ```suggestion
## Adoption
```

> Using authentik? We'd love to hear your story and feature your logo. Say hello at [hello@goauthentik.io](mailto:hello@goauthentik.io)!

Reviewer: ```suggestion
Using authentik? We'd love to hear your story and feature your logo. Email us at [hello@goauthentik.io](mailto:hello@goauthentik.io) or open a GitHub Issue/PR!
```
